src/components/app-controllers/SingleAppController/SingleAppController.js

```diff
@@ -36,6 +37,13 @@ export default createReactClass({
     return !(datafile.multi_year_mean == false && datafile.timescale == "monthly");
   },
 
+  //Returns metadata for datasets with thethe selected variable + scenario, any model.
+  //Passed as a prop for SingleDataController to generate model comparison graphs.
+  getModelContextMetadata: function () {
+    return _.where(this.state.meta,
+        {variable_id: this.state.variable_id, experiment: this.state.experiment});
+  },
+
```

Comment:
This is good. I'm not sure whether to prefer the ES6 native [Array.prototype.filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) over a more conveniently formulated underscore function. Your call.